src/internal/m365/service/teamschats/backup.go

```diff
@@ -145,9 +145,13 @@ func backupChats(
 		scope.Category().PathType().HumanString())
 	defer close(progressMessage)
 
+	qp := graph.QueryParams{
```

Comment:
this doesn't appear related to the rest of the PR. Could you please move it to a separate PR?

Reply:
Huh, weird.  that _is_ from a separate PR.  Not sure how it got caught up in this one.